app/response_processor.py

```diff
@@ -93,11 +93,11 @@ def process(self, msg, tx_id=None):
             self.logger.error(
                 "Invalid survey response received, skipping notification")
 
-        elif response_type.find("feedback") == -1:
+        elif self.send_to_downstream(response_type, decrypted_json):
```

Comment:
Do we need to pass all of the response_type and decrypted_json, when we only use the feedback and survey_id